announcement.md

```diff
@@ -2,7 +2,7 @@
 
 After a successful [inaugural session](inauguralsession.md) we aim to bring VVirtual Science Forum closer to its conceived goal: facilitate scientific communities organizing virtual events.
 
-In order to get a step closer  we are seeking for potential session organizers.
+In order to get a step closer to it we are seeking for potential session organizers.
 The necessary information to organize a session can be found [here](organizierguide.md).
 
-We will be grateful if this annoucement will be send around to colleagues that might be interested in organizing a virtual session.
+We will be grateful if this announcement will circulate around to colleagues that might be interested in organizing a virtual session.
```

Comment:
We would be grateful if you could circularte this announcement among colleagues that could be interested in organizing a virtual session.

Reply:
Address in latest commit